packages/ListBox/src/components/Content/Content.js

```diff
@@ -64,6 +64,7 @@ export default function Content(props) {
   return (
     <sc.PopoverContent
       {...getDOMAttributesForListBox({ idListBox, refLabel, ...state })}
+      {...moreProps}
```

Comment:
shouldn't this be before the getDOMAttr...?

we wouldn't want the consumer to override any specific declaration right?

Reply:
Ohhh, yes I see.